pandas/tests/io/test_html.py

```diff
@@ -77,10 +77,8 @@ def test_same_ordering(datapath):
 
 
 @pytest.mark.parametrize("flavor", [
-    pytest.param('bs4', marks=pytest.mark.skipif(
-        not td.safe_import('lxml'), reason='No bs4')),
-    pytest.param('lxml', marks=pytest.mark.skipif(
-        not td.safe_import('lxml'), reason='No lxml'))], scope="class")
+    pytest.param('bs4', marks=td.skip_if_no('lxml')),
```

Comment:
There are some of these in test_excel as well. Not sure if you meant to do all in this PR or not but if so worth a double check of other modules

Reply:
left them intentionally to avoid conflicts with your PRs.

have checked other modules, no other use of safe_import except one case of a test decorator where I think skip_if_no could be used directly. (change that in this PR?)

https://github.com/pandas-dev/pandas/blob/ee6b1318ff07806ead79dac8119ad9ce580afe1c/pandas/tests/io/test_gcs.py#L67-L69

and test_safe_import.